lib/workerholic/statistics_api.rb

```diff
@@ -0,0 +1,58 @@
+module Workerholic
+  class StatsAPI
```

Comment:
We should make the `Storage::RedisWrapper.new` a private class method and have the `StatsAPI` class be used directly, with class methods, not through an instance of it. It makes sense because it's an API, you don't need to keep instantiating it.